packages/wonder-blocks-form/src/components/text-field.tsx

```diff
@@ -222,22 +217,38 @@ class TextField extends React.Component<PropsWithForwardRef, State> {
         event,
     ) => {
         const {onFocus} = this.props;
-        this.setState({focused: true}, () => {
-            if (onFocus) {
-                onFocus(event);
-            }
-        });
+        if (onFocus) {
+            onFocus(event);
+        }
     };
 
     handleBlur: (event: React.FocusEvent<HTMLInputElement>) => unknown = (
         event,
     ) => {
         const {onBlur} = this.props;
-        this.setState({focused: false}, () => {
-            if (onBlur) {
-                onBlur(event);
-            }
-        });
+        if (onBlur) {
+            onBlur(event);
+        }
+    };
+
+    getStyles = (): (React.CSSProperties | Falsy)[] => {
```

Comment:
**nit:** You could also use `StyleType` from `wonder-blocks-core`.

```suggestion
    getStyles = (): StyleType => {
```

Reply:
This is what I needed! Thank you for pointing this out 😄

Reply:
I also updated this in TextArea since that's where I copied it from!